### ISSUE TITLE
Frontend: Adapt creation of new DI [Update]

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#49]
## User story
1. As a {HR-employee}
2. I want / need {to enter the name, surname and email-address for a new DI}
3. So that {the DI can be created successfully}

## Acceptance criteria
* update the "register"-page
  * fields: "name", "surname" and "email-address"
  * rename the page to: "Create new digital identity"

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
